appService.json

```diff
@@ -77,6 +77,9 @@
             "description": "Sends a OPTIONS request"
         }
     ],
+    "flowSteps":  [
+        "generic"
```

Comment:
Is it correct to call it "generic"? Can we use "http" instead?

Reply:
I would name it `HTTP call`.